File: src/main/java/com/github/weewar/mapviewer/dao/MapSearchCriteria.java
package com.github.weewar.mapviewer.dao;

public class MapSearchCriteria {
    private int firstElement;
    private int pageSize;

    public MapSearchCriteria setFirstElement(int firstElement) {
        this.firstElement = firstElement;
        return this;
    }

    public int getFirstElement() {
        return firstElement;
    }

    public MapSearchCriteria setPageSize(int pageSize) {
        this.pageSize = pageSize;
        return this;
    }

    public int getPageSize() {
        return pageSize;
    }
}

File: src/main/kotlin/com/github/weewar/mapviewer/dao/MapSearchCriteria.kt
package com.github.weewar.mapviewer.dao

class MapSearchCriteria {
    private var firstElement: Int = 0
    private var pageSize: Int = 0

    fun setFirstElement(firstElement: Int): MapSearchCriteria {
        this.firstElement = firstElement
        return this
    }

    fun getFirstElement(): Int {
        return firstElement
    }

    fun setPageSize(pageSize: Int): MapSearchCriteria {
        this.pageSize = pageSize
        return this
    }

    fun getPageSize(): Int {
        return pageSize
    }
}
